src/distr/other.rs

```diff
@@ -166,6 +167,7 @@ impl Distribution<u8> for Alphabetic {
 #[cfg(feature = "alloc")]
 impl SampleString for Alphanumeric {
     fn append_string<R: Rng + ?Sized>(&self, rng: &mut R, string: &mut String, len: usize) {
+        // SAFETY: `self` only samples alphanumeric characters, which are valid UTF-8.
```

Comment:
(Very low-priority since the preceding impl of `sample` pretty obviously does what you say here) Maybe add a `debug_assert!` that `self.sample_iter(rng).take(len)` returns UTF-8?

Reply:
Done